R/fct_dea-bootstrap.R

```diff
@@ -1,3 +1,6 @@
+#' @importFrom stats complete.cases median quantile rnorm sd var
```

Comment:
See comment above. I think we should rely on `::`

Reply:
See comments above.